flume/conf/flume.conf.template

```diff
@@ -29,5 +29,11 @@ CONFIG_FOLDER=/usr/cygnus/conf
 # Which is the config file
 CONFIG_FILE=/usr/cygnus/conf/cygnus.conf
 
-# Name of the agent
+# Name of the agent. This value should match with the begining of the keys in ${CONFIG_FILE} otherwise service not start.
```

Comment:
"Name of the agent. The name of the agent is not trivial, since it is the base for the Flume parameters naming conventions, e.g. it appears in <agent_name>.sources.http-source.channels=..."